docs/setting-up/client/mongodb.md

```diff
@@ -33,140 +64,213 @@ db.getSiblingDB("admin").createRole({
+1. Select *{{icon.cog}} Configuration-->{{icon.inventory}} PMM Inventory-->{{icon.addinstance}} Add Instance*.
+2. Select *MongoDB -- Add a remote instance*.
+3. Enter values for the fields.
+    - *Hostname*: The hostname or IP address of the node where the database server is running.
+    - *Service name*: Choose a service name.
+    - *Port*: Default is 27017 for MongoDB servers.
+    - *Username*: The username for connecting to the MongoDB database. Use the PMM user account if you chose to create one (`{{pmm_mongodb_user}}` in these examples).
+    - *Password*: The password for this user account. (`{{pmm_password}}` in these examples.)
+    - Additional options:
+        - *Use QAN MongoDB Profiler*: Activate if you want to use Query Analytics and have [set up profiling](#profiling).
 
-```sh
-service mongod restart
-```
+4. Click *Add service*.
 
-## Adding MongoDB Service Monitoring
+### On the command line
 
-Add monitoring as follows:
+Use `pmm-admin` to add the database server as a service using one of these example commands.
 
-```sh
-pmm-admin add mongodb --username=pmm --password=pmm
-```
+When successful, PMM Client will print `MongoDB Service added` with the service's ID and name. Use the `--environment` and `-custom-labels` options to set tags for the service to help identify them.
 
-where username and password are credentials for the monitored MongoDB access, which will be used locally on the database host. Additionally, two positional arguments can be appended to the command line flags: a service name to be used by PMM, and a service address. If not specified, they are substituted automatically as `<node>-mongodb` and `127.0.0.1:27017`.
+> **See also**: [`pmm-admin` man page][PMM_ADMIN_MAN_PAGE]
 
-The command line and the output of this command may look as follows:
+**Example**
 
 ```sh
-pmm-admin add mongodb --username=pmm --password=pmm mongo 127.0.0.1:27017
+pmm-admin add mongodb \
+--username={{pmm_mongodb_user}} --password={{pmm_password}} \
+--query-source=profiler --cluster=mycluster
 ```
 
-```
-MongoDB Service added.
-Service ID  : /service_id/f1af8a88-5a95-4bf1-a646-0101f8a20791
-Service name: mongo
+**Example**
+
+```sh
+pmm-admin add mongodb \
+--username={{pmm_mongodb_user}} --password={{pmm_password}} \
+mongo 127.0.0.1:27017
 ```
 
-Beside positional arguments shown above you can specify service name and service address with the following flags: `--service-name`, `--host` (the hostname or IP address of the service), and `--port` (the port number of the service). If both flag and positional argument are present, flag gains higher priority. Here is the previous example modified to use these flags:
+**Example**
 
 ```sh
-pmm-admin add mongodb --username=pmm --password=pmm --service-name=mongo --host=127.0.0.1 --port=27017
+pmm-admin add mongodb \
+--username={{pmm_mongodb_user}} --password={{pmm_password}} \
+--service-name=mymongosvc --host=127.0.0.1 --port=27017
```

Comment:
on netlify at least, some of these numbers (127 and 27017) show up as red...draws my eyes right to them but no idea why it is a different color if we intended them to be.

Reply:
Some weirdness with how it syntax highlights. I'll try some other content hints (or leave it out for plain text).

Reply:
Syntax highlighting is done by [Pygments](https://pygments.org/). The styling is built into the [Material theme for MkDocs](https://squidfunk.github.io/mkdocs-material/reference/code-blocks/). It seems numbers are just rendered that way according to the interpretation of the content (in this case, shell code).

While we can override the styling with CSS, there will eventually be someone else who doesn't like the replacement colour. IMHO a much simpler solution is to kindly ask your glazzies to refrain from being attracted to red things, and instead look at the code as a whole. I'm more in favour of keeping to platform defaults than overriding and customising.